src/components/registration-lite.js

```diff
@@ -245,16 +249,18 @@ const RegistrationLite = (
                     clearWidgetState();
                     return authErrorCallback(error);
                 }
+                setTicketDataLoaded(true);
                 setTicketTaxesError(true);
+            })
+            .finally(() => {
+                setTicketDataLoaded(true);
             });
     }
 
     // if we dont have yet ticket types and we didnt requested so far for them but we are already logged in
     // just dont render
     if(ticketTypes.length === 0 && !requestedTicketTypes && profileData) return null;
 
-    const allowedTicketTypes = ticketTypes.filter((tt) => (tt.sales_start_date === null && tt.sales_end_date === null) ||
```

Comment:
re add this
and change it to 
  const allowedTicketTypes = ticketTaxesLoaded ? ticketTypes.filter((tt) => (tt.sales_start_date === null && tt.sales_end_date === null) ||
        (nowUtc >= tt.sales_start_date && nowUtc <= tt.sales_end_date)): [] ;